File: swift/problem598.swift
class Solution {
    func maxCount(_ m: Int, _ n: Int, _ ops: [[Int]]) -> Int {
        if (ops.count == 0) {
            return m*n
        }
        
        var min_row = 40_001
        var min_col = 40_001
        
        for operation in ops {
            min_row = min(operation[0],min_row)
            min_col = min(operation[1],min_col)
        }
        
        return min_row*min_col
    }
}

File: python/problem598.py
class Solution:
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        if not ops:
            return n*m
        minx=ops[0][0]
        miny=ops[0][1]
        for i in ops:
            if i[0]<minx:
                minx=i[0]
            if i[1]<miny:
                miny=i[1]
        return minx*miny